src/Illuminate/Database/Console/DatabaseInspectionCommand.php

```diff
@@ -224,7 +224,7 @@ protected function ensureDependenciesExist()
     protected function installDependencies()
     {
         $command = collect($this->composer->findComposer())
-            ->push('require doctrine/dbal')
+            ->push('require doctrine/dbal^3.5.1')
```

Comment:
Don't we need a colon before the version?

Reply:
Yeah. That's right